cpu/cc2538/ldscripts/cc2538nf11.ld

```diff
@@ -25,6 +25,15 @@ MEMORY
     ram   (rwx) : ORIGIN = 0x20004000, LENGTH = 16K
 }
 
-INCLUDE cc2538.ld
+/* MCU Specific Section Definitions */
+SECTIONS
+{
+    .flashcca :
+    {
+        KEEP(*(.flashcca))
+    } > cca
+}
+
+INCLUDE cortexm_base.ld
```

Comment:
No, I'm not aware of any benefit. I just noticed that the `.flashcca` block is at the end of `SECTIONS` in *cc2538.ld*. And  I was curious about why you appended the include in a explicit position behind it.